Sources/MapboxDirections/Intersection.swift

```diff
@@ -243,10 +261,17 @@ extension Intersection: Codable {
         
         var lanes: [Lane]?
         if let approachLanes = approachLanes,
-            let usableApproachLanes = usableApproachLanes {
+            let usableApproachLanes = usableApproachLanes,
+            let preferredApproachLanes = preferredApproachLanes,
+            let usableLaneIndication = usableLaneIndication {
             lanes = approachLanes.map { Lane(indications: $0) }
+            // document assumptions and call them out in PR
             for i in usableApproachLanes {
                 lanes![i].isValid = true
+                lanes![i].validIndication = usableLaneIndication
```

Comment:
Just in case, should we avoid setting this indication as the active one if the lane doesn’t have this indication in the first place?

Reply:
I don't think it's possible to have a lane that is usable for a maneuver but doesn't have the indication that's usable for the maneuver.